lib/scrum/sprint_cleaner.rb

```diff
@@ -39,13 +39,19 @@ def remove_unplanned_label(card)
       card.remove_label(label) if label
     end
 
-    def move_cards(source_list)
+    def add_in_last_sprint_label(card, target_board_id)
+      @label = Trello::Label.create(name: 'In Last Spring', board_id: target_board_id)
```

Comment:
you should be using the name from the settings so that it can be customized, similarly as it is done with the waterline label: `@settings.label_names['in_last_sprint']`